src/test/java/jenkins/plugins/mailer/MailerJCasCCompatibilityTest.java

```diff
@@ -1,19 +1,20 @@
 package jenkins.plugins.mailer;
 
-import hudson.tasks.Mailer;
-import io.jenkins.plugins.casc.misc.RoundTripAbstractTest;
-import org.jvnet.hudson.test.RestartableJenkinsRule;
-
 import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertNotNull;
 import static org.junit.Assert.assertTrue;
 
+import org.jvnet.hudson.test.RestartableJenkinsRule;
+
+import hudson.tasks.Mailer;
+import io.jenkins.plugins.casc.misc.RoundTripAbstractTest;
+
```

Comment:
Import reorder could be mis-interpreted. Not a blocker here, as it's a test class.

Reply:
I moved from my work's laptop with IntelliJ to my personal laptop with Eclipse and I forgot not to mess with things unnecessarily. Sorry.